rskj-core/src/main/java/co/rsk/peg/union/UnionBridgeSupportImpl.java

```diff
@@ -81,36 +81,39 @@ public UnionResponseCode setUnionBridgeContractAddressForTestnet(@Nonnull Transa
         return UnionResponseCode.SUCCESS;
     }
 
-    private boolean isEnvironmentDisabled() {
+    private boolean isEnvironmentEnabled() {
         String currentNetworkId = constants.getBtcParams().getId();
 
-        boolean isEnvironmentDisable = currentNetworkId.equals(NetworkParameters.ID_MAINNET);
-        if (isEnvironmentDisable) {
+        boolean isMainnet = currentNetworkId.equals(NetworkParameters.ID_MAINNET);
+        if (isMainnet) {
             String baseMessage = String.format("Union Bridge Contract Address can only be set in Testnet and RegTest environments. Current network: %s", currentNetworkId);
-            logger.warn(LOG_PATTERN, "isEnvironmentDisable", baseMessage);
+            logger.warn(LOG_PATTERN, "isEnvironmentEnabled", baseMessage);
+            return false;
         }
-        return isEnvironmentDisable;
+        return true;
     }
 
-    private boolean isUnauthorizedCaller(Transaction tx) {
+    private boolean isAuthorizedCaller(Transaction tx) {
         AddressBasedAuthorizer authorizer = constants.getChangeAuthorizer();
-        boolean isUnauthorizedCaller = !authorizer.isAuthorized(tx, signatureCache);
-        if (isUnauthorizedCaller) {
+        boolean isAuthorized = authorizer.isAuthorized(tx, signatureCache);
+        if (!isAuthorized) {
             String baseMessage = String.format("Caller is not authorized to execute this method. Caller address: %s", tx.getSender());
-            logger.warn(LOG_PATTERN, "isUnauthorizedCaller", baseMessage);
+            logger.warn(LOG_PATTERN, "isAuthorizedCaller", baseMessage);
+            return false;
         }
 
-        return isUnauthorizedCaller;
+        return true;
     }
 
-    private boolean isInvalidAddress(RskAddress unionBridgeContractAddress) {
+    private boolean isValidAddress(RskAddress unionBridgeContractAddress) {
         // Check if the address is valid
-        boolean isInvalidAddress = unionBridgeContractAddress == null || unionBridgeContractAddress.equals(EMPTY_ADDRESS);
-        if (isInvalidAddress) {
+        boolean isValidAddress = unionBridgeContractAddress != null && !unionBridgeContractAddress.equals(EMPTY_ADDRESS);
+        if (!isValidAddress) {
             String baseMessage = "Union Bridge Contract Address cannot be null or empty";
-            logger.warn(LOG_PATTERN, "isInvalidAddress", baseMessage);
+            logger.warn(LOG_PATTERN, "isValidAddress", baseMessage);
+            return false;
         }
-        return isInvalidAddress;
+        return true;
```

Comment:
same here

Reply:
Updated. Now is returning the result of the validation variable.